src/router.rs

```diff
@@ -108,6 +109,35 @@ impl RouteInfo {
         }
         false
     }
+
+    /// Check if application contains matching route.
+    ///
+    /// This method does take `prefix` into account.
+    /// For example if prefix is `/test` and router contains route `/name`,
+    /// following path would be recognizable `/test/name` and `has_route_prefixed()` call
+    /// would return `true`.
+    pub fn has_route_prefixed(&self, path: &str) -> bool {
+        let path = if path.is_empty() {
+            "/".into()
+        } else if !self.router.prefix.is_empty() {
+            let pb = PathBuf::from(path);
```

Comment:
Use `Path::new()` instead, as argument is string slice, we can get `Path` directly without a need for allocation